src/dotenv/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.10.1"
+__version__ = "0.10.1"  # type: str
```

Comment:
Mypy should infer the type of `__version__` so it shouldn't be necessary to add a type hint for it.